src/ducks/balance/components/GroupPanel.jsx

```diff
@@ -93,6 +98,26 @@ class GroupPanel extends React.PureComponent {
     e.stopPropagation()
   }
 
+  componentDidUpdate(prevProps) {
+    // We have to revert optimisticExpanded if for example
+    // the network has failed
+    if (this.props.expanded !== prevProps.expanded) {
+      this.setState({ optimisticExpanded: this.props.expanded })
+    }
```

Comment:
It means that if I expand a panel it actually expands, but will automatically collapse few seconds after if the network failed ?

Reply:
You're right, and this is not actually the behavior we want. Since the data criticity is not high, I removed this bit of code. This is implemented (and tested 🎉 ) in https://github.com/cozy/cozy-banks/pull/980/commits/e53e6c9fd266d77fd07ba29351ae58ba7e70b123.